src/js/script.js

```diff
@@ -0,0 +1,10 @@
+var navMain = document.querySelector('.nav');
```

Comment:
At present, the `var` statement is used very rarely. You may encounter it in very rare cases or legacy code. Please use the `const` and `let` statements to declare variables.